src/main/postgres/service.ts

```diff
@@ -146,13 +146,27 @@ export function startPostgres(): ChildProcessWithoutNullStreams {
   return postgresql;
 }
 
-export async function shutdownPostgres(): Promise<void> {
-  console.info('Shutting down PostgreSQL...');
+function gracefulShutdown() {
   // On Windows, if we let the OS handle shutdown, it will not be graceful, and next start
   // is in recovery mode.
   // On Linux, `postgresql?.kill()` works fine, but the common `pg_ctl` is just as good.
-  execFile(path.join(postgresBinPath, 'pg_ctl'), ['-w', '-D', postgresConfig.dataDirectory, 'stop']);
-  return waitForPostgresqlStatus(ServiceStatus.Stopped);
+  execFile(path.join(postgresBinPath, 'pg_ctl'), ['-w', '-D', postgresConfig.dataDirectory, 'stop', '-m', 'fast']);
```

Comment:
The `-w` argument causes `pg_ctl` to wait 60 seconds for the server to finish shutting down, while we ignore the result of the operation and we wait only 5 seconds anyway. We should drop it, since it is useless.

Reply:
I don't remember exactly, why it was added in the first place, so let's see if there aren't any caveats (I'll retest on Windows to be sure)

Reply:
It will just wait for the shutdown to finish and return an error on timeout, which we don't check for anyway.